libs/core/src/components/sage-button/test/sage-button.spec.tsx

```diff
@@ -58,7 +58,22 @@ describe('sage-button', () => {
     const svg = root.shadowRoot.querySelector('svg');
     expect(svg).not.toBeNull();
   });
-});
-
-
 
+  it('clicking on sage-button from inside a form calls `handleClick` method', async () => {
```

Comment:
We need to fix the description of this spec. It should read as a sentence starting with what is in the `describe` function. 

So when we look at the whole test it starts off with the describe function in this case it is `describe('sage-button')`. So when we read the test cases (it functions). Using line 53 as an example; it would read `Sage Button renders icon button`. We could have improved that, but it still reads ok. 

I'll try to find a good article and update this comment and post it on our Confluence page. 

cc: @QuintonJason @anechol

Reply:
I like to read/write it in relation to the function: `it 'should/do/render/etc whatever action, show whatever UI'`